src/scripts/creature-stats.js

```diff
@@ -554,12 +564,36 @@ function getCustomActionsData() {
 }
 
 function getCustomLegActionsData() {
```

Comment:
This is something that's probably better to pair on in-person as opposed to asynchronously, but this is a very meaty function that might be better split into smaller chunks of logic — especially when it comes to some of these nested `for` loops.